.github/actions/lint/action.yml

```diff
@@ -3,5 +3,7 @@ description: 'Check links to external sites redirect to a new tab'
 runs:
   using: "composite"
   steps:
+    - run: pip install PyYAML
+      shell: bash
     - run: python3 $GITHUB_ACTION_PATH/lint_links.py
       shell: bash
```

Comment:
You can run [multi-line commands](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun)